jax/experimental/jax2tf/tests/call_tf_test.py

```diff
@@ -65,6 +66,34 @@ def _named_test(**kwargs):
 
 class CallTfTest(tf_test_util.JaxToTfTestCase):
 
+  @classmethod
+  def setUpClass(cls):
+    # Pick one device from each available platform
+    cls.jax_platforms = []
+    for backend in ["cpu", "gpu", "tpu"]:
+      try:
+        devices = jax.devices(backend)
+      except RuntimeError:
+        devices = []
+      if devices:
+        cls.jax_platforms.append(devices[0].platform)
+
+    # One TF device of each device_type
+    cls.tf_devices = []
+    for tf_device in (tf.config.list_logical_devices("TPU") +
+                      tf.config.list_logical_devices("GPU") +
+                      tf.config.list_logical_devices()):
+      if tf_device.device_type == "TPU_SYSTEM":
+        continue  # A virtual device
+      if all(tf_device.device_type != d.device_type for d in cls.tf_devices):
+        cls.tf_devices.append(tf_device)
+
+    # Find platforms that are available for both JAX and TF
+    cls.jax_and_tf_platforms = (
+      set(cls.jax_platforms).intersection(
```

Comment:
It could be more readable to use the infix `&` operator, instead of calling `.intersection()`.

Reply:
Done